llvm_util/utils.cpp

```diff
@@ -332,6 +336,23 @@ class llvm2alive_ : public llvm::InstVisitor<llvm2alive_, unique_ptr<Instr>> {
     if (!fn) // TODO: support indirect calls
       return error(i);
 
+    if (llvm::isMallocLikeFn(&i, &TLI, false) && fn->getName() == "malloc") {
```

Comment:
uhm, why the name check?

Reply:
Because isMallocLikeFn and isFreeCall returns true on variants such as `new` / `delete` operators in C++ as well.
It is conservatively checked because I couldn't find from C++ standard whether `new` should always use the `malloc` call provided.

Or, what do you think about enabling this with a command line flag so people can assume that they are equivalent if needed?

Reply:
I think it's fine. `isMallocLikeFn` should only return true for functions where the first argument is the size and that allocate memory.